core/api/pod-api/src/fs.ts

```diff
@@ -28,7 +28,7 @@ export interface Stats {
 export interface FS {
     readFile(path: string, options: EncodingOptions): Promise<string>;
     readFile(path: string): Promise<Uint8Array>;
-    writeFile(path: string, content: string, options: EncodingOptions): Promise<void>;
+    writeFile(path: string, content: string, options?: EncodingOptions): Promise<string>;
```

Comment:
Besides what I say above, change of interfaces so deep down in the polyPod... I created this idea https://jira.polypoly.eu/browse/PROD4POD-957 so it's something we might want to consider in the future.